script/releases/release-template/2-enqueueTimelock.s.sol

```diff
@@ -51,7 +54,8 @@ contract Queue is MultisigBuilder {
         return _opsCalls;
     }
 
-    function zeusTest() public override {
+    function testDeploy() public virtual {
         // Test function implementation
+        _execute();
```

Comment:
Can we call this internal function `run` or something? `execute` can be confused with the multisig execute.

Reply:
_execute() is an override function, gonna push back on that. you shouldn't be confused while overriding something that has documentation IMO